src/chrome/content/rules/KSH.xml

```diff
@@ -9,4 +9,9 @@
 
 	<rule from="^http://(statinfo|elektra)\.ksh\.hu/"
 		to="https://$1.ksh.hu/" />
+	
```

Comment:
Please add a comment about the issue and move the exclusion above the `<rule>` block.